numpy/core/setup.py

```diff
@@ -24,6 +24,10 @@
 NPY_RELAXED_STRIDES_DEBUG = (os.environ.get('NPY_RELAXED_STRIDES_DEBUG', "0") != "0")
 NPY_RELAXED_STRIDES_DEBUG = NPY_RELAXED_STRIDES_DEBUG and NPY_RELAXED_STRIDES_CHECKING
 
+# Set NPY_DISABLE_SVML=1 in the environment to disable the vendored SVML
+# library. This option only has significance on Linux x86_64.
```

Comment:
```suggestion
# library. This option only has significance on Linux x86_64 when 
# cross-compiling for a non-x86 platform.
```

Reply:
Does this make it more clear?

Reply:
I clarified the text along these lines and rebased on the latest main branch.

Reply:
Looks good. Is "cross-build" more commonly used? In any case, this documentation should be copied to a new section (cross-compiling or cross-building) in `doc/source/user/building.rst`. If you wish it could be done in a follow on PR (maybe even by someone else), in that case please open a DOC issue and I will merge this as-is. If there is any other information that would be helpful to people trying to cross-build, please add it to the issue/section.